.github/workflows/action-pypi-build-and-deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: PyPi Build and Deploy 🐍📦
+
+on:
+  release:
+    types: [published]
+  # use this for testing
+  push:
+    branches:
+      - master
+
+jobs:
+  build-n-publish:
+    name: Build and publish ESMValTool on PyPi
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
```

Comment:
We support python3.6+, so should we build with 3.6?

Reply:
the actual build and deployment is done with the latest Python, so no need to test with other older versions :+1:

Reply:
also - that would mean we'd have to release three different packages `_py36.., _py37, _py38...` - we could do that but I reckon it's too much since we'd confuse the users even more :grin: ...